### DISCUSSION TITLE
The "sort_as" attribute is not working in references.yaml

### DISCUSSION THREAD

**Discussion by zsofiaj** [thegetty/quire#737] [Get Help] [answered]
I have several names in my references.yaml that are not sorting correctly per the automatic alphabetization on my bibliography page. For example, "van Marle" is sorting under V, but I want it to sort under M. I believe this was working in Quire v0, but I'm not completely sure. Here are some screenshots of how the citations appear on the page, and how I've entered them in references.yaml. Am I doing something wrong, or is this a bug? 

<img width="1424" alt="Screen Shot 2023-05-18 at 1 11 25 PM" src="https://github.com/thegetty/quire/assets/68071544/5de874d6-c52a-4a4d-af91-d3afdc54340c">
<img width="983" alt="Screen Shot 2023-05-18 at 1 12 42 PM" src="https://github.com/thegetty/quire/assets/68071544/59e419ae-b04b-4d48-8517-a8f84c51a806">

**Comment by Erin-Cecele** [comment]
Hi @zsofiaj, Quire may not be sorting properly because you have two identical `sort_as` values. Try changing them to `sort_as: Marle 1920` and `sort_as: Marle 1923`. See if that helps.

**Reply by geealbers** [reply, marked as answer]
Yes, @Erin-Cecele is correct that you'll want the `sort_as` values to be unique. Otherwise Quire could sort all the "Marles" in different orders every time you preview/build the project. And the use of years along with the name, as Erin suggests, is a good solution.

There is **also** though a bug in how Quire is handling the `sort_as` values. You can fix it by going to `_plugins/filters/sortReferences.js` in your project, and replacing lines 34 and 35 with these two lines, which you'll see are very similar to what's there, except for an extra set of `sort_as` options in the middle:

```js
let a = sortById ? itemA.sort_as || itemA.id : itemA.sort_as || itemA.full
let b = sortById ? itemB.sort_as || itemB.id : itemB.sort_as || itemB.full
```

**Reply by zsofiaj** [reply]
Thanks so much, @geealbers and @Erin-Cecele! This worked great. I added dates to make all of the sort values unique (wish I had thought of that), and I replaced those two lines in the sortReferences.js file.
